lib/server.js

```diff
@@ -189,3 +203,21 @@ module.exports = {
   }
 };
 
+function cleanUpAndExit() {
+  if (electronPids) {
```

Comment:
I get that.

What I'm saying is - when the following 7 lines of  code was added (no other code from your patch), it was able to terminate the app gracefully.

```
function cleanUpAndExit() {
  console.log('=====>>>>> cleanUpAndExit');
  setTimeout(function() {
    process.exit(0);
  }, 1000);
}
process.on('SIGINT', cleanUpAndExit);
process.on('SIGTERM', cleanUpAndExit);
```
Are you seeing the same behavior?